src/main/java/ch/njol/skript/sections/ConditionalSection.java

```diff
@@ -0,0 +1,190 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.sections;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.lang.util.CodeSection;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings("NotNullFieldNotInitialized")
+public class ConditionalSection extends CodeSection {
+
+	static {
+		Skript.registerSection(ConditionalSection.class,
+			"else",
+			"else if <.+>",
+			"[(1¦if)] <.+>");
+	}
+
+	private enum ConditionalType {
+		ELSE, ELSE_IF, IF
+	}
+
+	private ConditionalType type;
+	private Condition condition;
+
+	private Kleenean hasDelayAfter;
+
+	@Override
+	public boolean init(Expression<?>[] exprs,
```

Comment:
Can all of the parameters go on one line like in `LoopSection`, or should they be split up there too?